sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -65,10 +65,12 @@ For example, `https://<my-custom-subdomain>.cognitiveservices.azure.com/`.
 Install the Azure Text Analytics client library for Python with [pip][pip]:
 
 ```bash
-pip install azure-ai-textanalytics
+pip install azure-ai-textanalytics --pre
 ```
 
-> Note: This version of the client library supports the v3.0 version of the Text Analytics service
+> Note: This version of the client library supports both the v3.0 version, and the preview versions of the v3.1 version of the Text Analytics service. If you do not specify the `--pre` flag when installing the package, you will download version 5.0.0 of the library, which will only
+include support of the services v3.0 version. Specifying the `--pre` flag will give you the latest beta release of this library, which includes
+support of all preview versions of v3.1, and also continuing support of the service's v3.0 version.
```

Comment:
this is very nit picky of me, but maybe in the future this could be conveyed better in a table that shows what the command is, the version of the client library you get, and the version of the service supported. As we support more versions, I'm concerned this note will get complicated

Reply:
I think that at most there will be two SDK versions that we talk about in the readme though (the latest GA version and the latest beta version), and for each SDK version, we really only need to mention one API version corresponding (i.e., this supports up to version xx). After typign all this out, I do think a table will look clean thoughk, so I'll make it one now